script/bootstrap-st2express

```diff
@@ -3,6 +3,10 @@
 
 PROJECT_ROOT=/opt/puppet
 
+if [ $UID -ne 0 ]; then
```

Comment:
Are you sure you want to use `$UID` instead of `$EUID` as http://askubuntu.com/a/30157 suggests?
As I understood it's less error-prone.